app/src/lib/menu-update.ts

```diff
@@ -199,7 +202,6 @@ function getRepositoryMenuBuilder(state: IAppState): MenuStateBuilder {
     'create-commit',
     'show-history',
     'show-branches-list',
-    'open-external-editor',
```

Comment:
Why have we removed this? This list of menu IDs is the ones to hide when a repository is missing from disk, and currently `Open in External Editor` remains enabled:

<img width="347" src="https://user-images.githubusercontent.com/359239/40208256-0e7e0314-5a7c-11e8-92c7-a07ae2eb02f8.png">